gcc4mbed/external/mbed/libraries/mbed/targets/hal/TARGET_NXP/TARGET_LPC43XX/analogin_api.c

```diff
@@ -25,110 +28,145 @@
 #define ANALOGIN_MEDIAN_FILTER      1
 
 static inline int div_round_up(int x, int y) {
-  return (x + (y - 1)) / y;
+	return (x + (y - 1)) / y;
```

Comment:
I was using eclipse to compile and auto-indent. It could have change the indentation of some portions of the code. I will check my settings and avoid to use auto-indentation.